### PR TITLE
REF: Use In-App browser on iOS

### PR DESCRIPTION
**Pull request by marcosrdz**

### CODE REVIEW

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/zte6e36zy61azk5dtt9cm2qfpg

[download apk](http://filestorage.bluewallet.io:1488/1599268696.apk)

**Review by Overtorment — CHANGES_REQUESTED** [review]
this did not solve apple pay on ios, and I don't see a reason to merge this.

File: screen/wallets/transactions.js [hunk 1]
```diff
@@ -655,6 +646,27 @@ export default class WalletTransactions extends Component {
     }
   };
 
+  navigateToBuyBitcoin = async () => {
+    const uri = await BuyBitcoin.generateURL(this.state.wallet);
+    if (getSystemName() === 'Mac OS X') {
+      InAppBrowser.isAvailable()
+        .then(_value => InAppBrowser.open(uri, { dismissButtonStyle: 'done' }))
```
Comment (Overtorment) [inline screen/wallets/transactions.js:653]: can you hide all this InAppBrowser business inside buy-bitcoin screen? that's the only purpose of that screen,
and its bad that all associated complexity is leaking outside (in transactions screen in this case). what if we will decide to place 'buy bitcoin' button in other places? we would have to copy-paste all this logic

**Comment by GladosBlueWallet** [comment]
♫ This was a triumph. I'm making a note here: HUGE SUCCESS ♫

 [android in browser] https://appetize.io/app/p4nmwhamzne2h04g547utub414

[download apk](http://filestorage.bluewallet.io:1488/1599505312.apk)

File: screen/lnd/scanLndInvoice.js [hunk 2]
```diff
@@ -438,7 +438,7 @@ export default class ScanLndInvoice extends React.Component {
                       <ActivityIndicator />
                     </View>
                   ) : (
-                   <View style={styles.payButtonContainer}>
+                    <View style={styles.payButtonContainer}>
```
Comment (Overtorment) [inline screen/lnd/scanLndInvoice.js:441]: wtf

File: screen/wallets/transactions.js [hunk 3]
```diff
@@ -655,6 +645,10 @@ export default class WalletTransactions extends Component {
     }
   };
 
+  navigateToBuyBitcoin = () => {
+    BuyBitcoin.navigate(this.state.wallet);
```
Comment (Overtorment) [inline screen/wallets/transactions.js:649]: good enough

Comment (Overtorment) [reply screen/wallets/transactions.js:649]: I hope its not breaking some RN best practices

**Review by Overtorment — APPROVED** [review]
looks much cleaner now. lets distribute & test build and merge

**Review by ncoelho — APPROVED** [review]

File: screen/lnd/scanLndInvoice.js (hunk 2 above)
Comment (marcosrdz) [reply screen/lnd/scanLndInvoice.js:441]: Lint. Single space

**Pull request merged**